pyproject.toml

```diff
@@ -60,6 +60,7 @@ extend-include = ["*.ipynb"]
 [tool.ruff.lint]
 ignore = [
     "PERF203", # Incorrect detection
+    "S608", # TODO: fix possible injections in CQL ?
```

Comment:
Should we sanitize keyspace, node_table, targets_table before passing to query ?

Reply:
Added sanitization.